src/Glamour-Morphic-Renderer/GLMMorphicSmalltalkCodeRenderer.class.st

```diff
@@ -87,7 +87,7 @@ GLMMorphicSmalltalkCodeRenderer >> textModelFor: aPresentation [
 { #category : #rendering }
 GLMMorphicSmalltalkCodeRenderer >> textMorphForModel: aTextModel [
 	| morph  |
-	morph := GLMPluggableTextMorph new.
+	morph := RubPluggableTextMorph new.
 	morph styler workspace: aTextModel.
```

Comment:
So where is the styler?

Reply:
We will integrate this and you can open another issue. We cannot continue to have large changes in the air.

Reply:
> So where is the styler? Does that depend on the DNU?

@StephanEggermont, was this a side question? or a concern about this change?

Reply:
There have been other issues where the styler is missing, breaking functionality, so this might break something more in the image. I expressed my concern already on the ML